pom.xml

```diff
@@ -12,54 +12,63 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>1.3.3.RELEASE</version>
+        <version>1.4.1.RELEASE</version>
     </parent>
 
     <properties>
         <java.version>1.8</java.version>
         <opencga.version>0.5.2</opencga.version>
         <biodata.version>0.4.5</biodata.version>
         <cellbase.version>3.1.3</cellbase.version>
+        <postgresql.version>9.1-901-1.jdbc4</postgresql.version>
```

Comment:
We need to keep the PostgreSQL dependency because we are using it for the job repository database, but we could let Spring manage the driver version.